src/containers/changePassword/styles.module.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../scss/mixins/input.scss';
+@import '../../scss/shared/colors.scss';
+@import '../../scss/mixins/button.scss';
+@import '../../scss/shared/fonts.scss';
+@import '../../scss/shared/breakpoints.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
```

Comment:
We always have header and footer on page. So maybe it's better to do smth like this:
height: calc(100vh - Header height - Footer height)